Lib/ctypes/test/test_values.py

```diff
@@ -73,15 +73,17 @@ class struct_frozen(Structure):
                 self.assertTrue([entry.code[i] for i in range(abs(entry.size))])
                 # Check the module's package-ness.
                 with import_helper.frozen_modules():
-                    spec = importlib.util.find_spec(modname)
+                    # Hide the message written by the __hello__ module.
+                    with captured_stdout():
+                        spec = importlib.util.find_spec(modname)
```

Comment:
I'm wondering if it wouldn't be useful to just not print anything from the `__hello__` module unless it's invoked as a `__main__` module? That could possibly allow us to remove a lot of captured_stdout() calls. What is this output actually needed for (other than tradition)?

Reply:
I'm sure we didn't think about it when the module was added.